Content.Server/Station/Systems/StationSystem.cs

```diff
@@ -288,6 +314,21 @@ public Filter GetInStation(StationDataComponent dataComponent, float range = 32f
         return filter;
     }
 
+    //SL start
+    public EntityUid InitializeNewStationMidRound(EntityUid gridId, EntProtoId stationProtoId)
+    {
+        var station = EntityManager.SpawnEntity(stationProtoId, MapCoordinates.Nullspace);
+        var data = Comp<StationDataComponent>(station);
+        var name = MetaData(station).EntityName;
+        AddGridToStation(station, gridId, null, data, name);
+        // so far can't find any instance where anything that listens for this not running would cause the entire game server to shit and die
```

Comment:
but we also want our code to behave as expected, and be futureproofed for changes in the future where this might be the case

why isn't it being raised? does it interfere with the currently existing station?